src/Runner.Worker/ExecutionContext.cs

```diff
@@ -105,6 +105,8 @@ public interface IExecutionContext : IRunnerService
         // others
         void ForceTaskComplete();
         void RegisterPostJobStep(IStep step);
+        IStep RegisterCompositeStep(IStep step, DictionaryContextData inputsData);
```

Comment:
This is a bit of a misnomer. The "composite step" is the outer thing. Here we're an individual inner step.

Reply:
"InsertNestedStep"?